package.json

```diff
@@ -44,5 +43,8 @@
   "bugs": {
     "url": "https://github.com/scottie1984/swagger-ui-express/issues"
   },
-  "homepage": "https://github.com/scottie1984/swagger-ui-express"
+  "homepage": "https://github.com/scottie1984/swagger-ui-express",
+  "dependencies": {
+    "swagger-ui-dist": "^3.17.1"
```

Comment:
can be updated to 3.17.5

Reply:
@kibertoad, it could be, but it doesn't need to be - the `^` indicates that any version in the `3.*.*` range can be installed, and the highest version will always be selected.

Reply:
Only if package-lock wasn't yet generated :)

Reply:
@kibertoad, my understanding is that dependents will always get the highest in-range version, regardless of whether a lockfile is in the project, since `package-lock.json` is never published to the registry 🤔